seqr/utils/elasticsearch/constants.py

```diff
@@ -351,10 +351,13 @@ def get_prediction_response_key(key):
 SHARED_QUALITY_FIELDS = {'gq': 5}
 SNP_QUALITY_FIELDS = {'ab': 5}
 SNP_QUALITY_FIELDS.update(SHARED_QUALITY_FIELDS)
+MITO_QUALITY_FIELDS = {'hl': 5}
+MITO_QUALITY_FIELDS.update(SHARED_QUALITY_FIELDS)
 SV_QUALITY_FIELDS = {'qs': 10}
 SV_QUALITY_FIELDS.update(SHARED_QUALITY_FIELDS)
 QUALITY_QUERY_FIELDS.update(SNP_QUALITY_FIELDS)
 QUALITY_QUERY_FIELDS.update(SV_QUALITY_FIELDS)
+QUALITY_QUERY_FIELDS.update(MITO_QUALITY_FIELDS)
```

Comment:
Don't you also need to update the `POPULATIONS` constant to support the new frequency filter?

Reply:
The `POPULATIONS` has been updated for displaying the mito frequencies. No more updates are required for the frequency filters.